common/str.h

```diff
@@ -306,6 +307,52 @@ class String {
 	 */
 	static String vformat(const char *fmt, va_list args);
 
+	/** Finds the index of a character in the string */
+	size_t find(char c, size_t pos = 0) const;
+
+	/** Does a find for the passed string */
+	size_t find(const char *s) const;
+	uint32 find(const String &str, uint32 pos = 0) const;
+
+	/** Does a reverse find for the passed string */
+	size_t rfind(const char *s) const;
+	size_t rfind(const String &s) const {
+		return rfind(s.c_str());
+	}
+
+	/** Does a reverse find for a passed character */
+	size_t rfind(char c, size_t pos = npos) const;
+
+	/** Find first character in the string matching the passed character */
+	size_t find_first_of(char c, size_t pos = 0) const;
```

Comment:
The existing methods in this class use a camel case naming convention.

Reply:
Good point. Those methods currently match the naming of methods from the C std::string class, but for ScummVM it'll be better for consistency to have them camelcase. Particularly since for both the current Ultima code as well as any future code we import that uses std::string, it will be pretty simple to do a global search and replace to correct the method names to the ScummVM version